Formula/zrok.rb

```diff
@@ -0,0 +1,42 @@
+require "language/node"
+
+class Zrok < Formula
+  desc "Geo-scale, next-generation sharing platform built on top of OpenZiti"
+  homepage "https://zrok.io"
+  url "https://github.com/openziti/zrok.git",
+    tag:      "v0.3.5",
+    revision: "917891e9b1fbe28e112ff7e399a92d60080c79e4"
```

Comment:
```suggestion
  url "https://github.com/openziti/zrok.git",
      tag:      "v0.3.5",
      revision: "917891e9b1fbe28e112ff7e399a92d60080c79e4"
```
Align the arguments to `url`.